tools/clang/tools/dxa/dxa.cpp

```diff
@@ -281,16 +286,32 @@ void DxaContext::ListParts() {
 using namespace hlsl::options;
 
 int __cdecl main(int argc, _In_reads_z_(argc) char **argv) {
+  if (llvm::sys::fs::SetupPerThreadFileSystem())
+    return 1;
+  llvm::sys::fs::AutoCleanupPerThreadFileSystem auto_cleanup_fs;
+  if (FAILED(DxcInitThreadMalloc())) return 1;
```

Comment:
maybe here too? though this rarely fails. Maybe true of the above as well?

Reply:
Something seriously wrong if this fails, again might crash if we try to print.